### ISSUE TITLE
I try to reproduce the results of your experiment about RMSE.

### ISSUE THREAD

**Issue by DavidZhang88** [SimiaoZuo/Transformer-Hawkes-Process#2]
Dear Zuo，
I hope everything is fine with you, after saw your paper, I am really interested in you work, and learned a lot from it. However, I have a little question, how do you get the so small RMSE?  While I get the RMSE on Financal data, is like
`Minimum RMSE:  xx.xxxxx` while in the paper, it is only 0.93.
Are there some hyper-parameters about loss function need to adjustment?  I set
 `loss =  0 * event_loss + 0* pred_loss + se / scale_time_loss`
But it still doesn' t work, could you offer me some help? 
Thank you.

Sincerely yours,
Luning Zhang

**Comment by DavidZhang88** [comment]
The result in your paper,  the unit should be minite, and in code is second right? I think I figure it out. Thank you again for the efficient and easy-to-understand code. I learned a lot from it.

**Comment by ritvik06** [comment]
It is difficult to reproduce the results without knowing the scale of normalisation of the datasets. Have you been able to reproduce the results given in the paper? If yes, then at what alterations/normalisation did you have to do?

**Comment by ritvik06** [comment]
Another issue, as the mark and time prediction loss is added to the log likelihood term as part of our training, one would need to scale these losses to make them comparable. 

Although I figured the scale of their normalisation for the RMSE results by referring to the results for the used datasets given in the other paper for eg. RMTPP (Du et al.) and Neural Hawkes Process etc.
